aerogear/mobile-developer-console
# WebSockets - keep UI updated

# Motivation

https://issues.jboss.org/browse/AEROGEAR-7948

# Verification steps

- setup OpenShift with mobile-core
- deploy Mobile CI/CD service
- build and run MDC
- open more browser windows with MDC
- in one of them create mobile apps
- new mobile apps should appear in other windows without refresh
- create buildConfig in OpenShift
- it should appear in MDCs
- in one MDC instance trigger build
- it should appear in other windows
- as the build progresses, UI should be updated
